### DISCUSSION TITLE
Schedule based activation

### DISCUSSION THREAD

**Discussion by DennisDyallo** [darkreader/darkreader#6548] [Q&A] [answered]
First of all, loving the app. Been using now for about 6 months in both FF and Brave without any issues.

Feature Suggestion:  schedule based activation.

Let me be able to set a time for when the dark mode activates. Would help me a lot not having to toggle the on/off state manually.

Hopefully it can be done.

Thanks.

**Comment by bershanskiy** [comment, marked as answer]
This feature is called "Automation". To use it: Click on Dark Reader icon -> Click on dots next to ON/OFF switch on popup -> set time and make sure corresponding checkbox is cheked. Does this help?

**Reply by DennisDyallo** [reply]
Yes! Thank you I didn't see the menu options :)
